math/binarymath/arithmeticmean_test.go

```diff
@@ -0,0 +1,64 @@
+// arithmeticmean_test.go
+// description: Test for Arithmetic mean
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see arithmeticmean.go
+package binarymath
+
+import "testing"
+
+func TestArithmeticMean1(t *testing.T) {
+	tests := getTests()
+	for _, tv := range tests {
+		t.Run(tv.name, func(t *testing.T) {
+			result := ArithmeticMean1(tv.a, tv.b)
+			if result != tv.result {
+				t.Errorf("Wrong result! Expected:%d, returned:%d ", tv.result, result)
+			}
+		})
+	}
+
+}
+
+func TestArithmeticMean2(t *testing.T) {
+	tests := getTests()
+	for _, tv := range tests {
+		t.Run(tv.name, func(t *testing.T) {
+			result := ArithmeticMean2(tv.a, tv.b)
+			if result != tv.result {
+				t.Errorf("Wrong result! Expected:%d, returned:%d ", tv.result, result)
+			}
+		})
+	}
+
+}
+func getTests() []struct {
+	name   string
+	a      int
+	b      int
+	result int
+} {
+	var tests = []struct {
+		name   string
+		a      int
+		b      int
+		result int
+	}{
+		{"(2+4)/2=3", 2, 4, 3},
+		{"(5+5)/2=5", 5, 5, 5},
+		{"(1000+1002)/2=1000", 1000, 1002, 1001},
+		{"(80+40)/2=60", 80, 40, 60},
+		{"(7+4)/2=5", 7, 4, 5},
```

Comment:
write the names instead of calculations - anyone reading the code already knows that they are averages so makes the "(a + b)/2" redundant.
```suggestion
		{"Average of 2 and 4", 2, 4, 3},
		{"Average of 5 and 5", 5, 5, 5},
		{"Average of 1000 and 1002", 1000, 1002, 1001},
		{"Average of 80 and 40", 80, 40, 60},
		{"Average of 7 and 4", 7, 4, 5},
```

Reply:
Done